subsys/bluetooth/mesh/blob_srv.c

```diff
@@ -851,6 +856,7 @@ static int blob_srv_init(const struct bt_mesh_model *mod)
 {
 	struct bt_mesh_blob_srv *srv = mod->rt->user_data;
 
+	srv->chunk_size = CHUNK_SIZE;
```

Comment:
As I mentioned above, `chunk_size` field is redundant. It is possible just to use macro.